backend/app/api/v1/endpoints/ocp/ocpJobs.py

```diff
@@ -12,7 +12,7 @@
 
 @router.get('/api/v1/ocp/jobs',
             summary="Returns a job list",
-            description="Returns a list of jobs in the specified dates. \
+            description="Returns a list of jobs in the specified dates of requested size. \
```

Comment:
I think you should remove the "of requested size" ... the `size` and `offset` query parameters are self-describing, and this makes the sentence unnecessarily awkward.

Reply:
Removed the unnecessary phrase.